lib/twirp/encoding.rb

```diff
@@ -17,13 +17,18 @@ module Twirp
 
   module Encoding
     JSON = "application/json"
+    # An opt-in content type useful when curling or manually testing a twirp
+    # service.  This will fail if unknown fields are encountered. The return
+    # content type will be application/json.
+    JSON_STRICT = "application/json; strict=true"
     PROTO = "application/protobuf"
 
     class << self
 
       def decode(bytes, msg_class, content_type)
         case content_type
         when JSON  then msg_class.decode_json(bytes, ignore_unknown_fields: true)
+        when JSON_STRICT then msg_class.decode_json(bytes)
```

Comment:
let's add `ignore_unknown_fields: false` explicitly, so it is more clear what is the difference with JSON

Reply:
good idea, done.